.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.1
```

Comment:
Sounds good — better not to tie to a particular version — updated PR on the way.